### PR TITLE
navbar

### PR DESCRIPTION
**Pull request by fadeomar**
#42

### CODE REVIEW

File: src/components/Navbar/HeaderNav.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  const classes = useStyles();
+  return (
+    <Grid container justify="flex-end" spacing={6}>
+      <Grid item className={classes.gridItem}>
+        <OpendItem
+          name="Advisory"
+          items={[
+            "Strategy & Governance",
+            "Stakeholder Engagement",
+            " Materiality Assessment",
```
Comment (JoseChungFOS) [inline src/components/Navbar/HeaderNav.js]:
```suggestion
            "Materiality Assessment",
```

File: src/components/Navbar/HeaderNav.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            "Impact Management & Metrics",
+            " Reporting & Communications",
```
Comment (JoseChungFOS) [inline src/components/Navbar/HeaderNav.js]:
```suggestion
            "Reporting & Communications",
```

File: src/components/Navbar/HeaderNav.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+            "Frameworks & Standards",
+            "Program Management",
+            "Training & Capacity Building"
+          ]}
+        />
+      </Grid>
+      <Grid item className={classes.gridItem}>
+        <OpendItem
+          name="standards"
```
Comment (JoseChungFOS) [inline src/components/Navbar/HeaderNav.js]:
```suggestion
          name="Standards"
```

File: src/components/Navbar/HeaderNav.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+          items={["AA1000AP", "AA1000SES", "AA1000AS"]}
+        />
+      </Grid>
+      <Grid item className={classes.gridItem}>
+        <OpendItem
+          name="insights"
```
Comment (JoseChungFOS) [inline src/components/Navbar/HeaderNav.js]:
```suggestion
          name="Insights"
```

File: src/components/Navbar/HeaderNav.js [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+        />
+      </Grid>
+      <Grid item className={classes.gridItem}>
+        <Link to="/case-studies" className={classes.navLink}>
+          <Typography>Case Studies</Typography>
+        </Link>
+      </Grid>
+      <Grid item className={classes.gridItem}>
+        <OpendItem
+          name="about"
```
Comment (JoseChungFOS) [inline src/components/Navbar/HeaderNav.js]:
```suggestion
          name="About"
```

File: src/components/Navbar/HeaderNav.js [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+          items={[
+            "History ",
+            "Advisory Board",
+            "Management Team",
+            "Partners",
+            "Careers"
+          ]}
+        />
+      </Grid>
+      <Grid item className={classes.gridItem}>
```
Comment (JoseChungFOS) [inline src/components/Navbar/HeaderNav.js]: Couple of things here:
* These will be external links so we can't use the `Link` component. It'll just be a simple anchor link 
  * `a href="https://www.linkedin.com/company/138636" target="_blank"`
  * `a href="https://twitter.com/AAInsights" target="_blank"`
* Also the icons don't seem to be aligned - I can't quite figure out why that is, but if they could be bottom-aligned that would be great

File: src/components/Navbar/index.js [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+import AcountabilityLogo from "../../img/logo.svg";
+
+import HeaderNav from "./HeaderNav";
+import useStyles from "./style";
+
+export default function ButtonAppBar() {
+  const classes = useStyles();
+
+  return useMediaQuery("(min-width:1233px)") ? (
```
Comment (JoseChungFOS) [inline src/components/Navbar/index.js]: This breakpoint is too wide - ideally we'd set it the the absolute minimum before spacing between items is too narrow. Maybe around 900px if possible?

File: src/components/Navbar/mobile.js [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+const ModileHeader = () => {
+  const classes = useStyles();
+  const navItems = [
+    { name: "Advisory", link: "/advisory" },
+    { name: "Standards", link: "/standards" },
+    { name: "Insights", link: "/insights" },
+    { name: "Case Studies", link: "/case-studies" },
+    { name: "About", link: "/about" },
+    { name: "partners", link: "/Partners" },
```
Comment (JoseChungFOS) [inline src/components/Navbar/mobile.js]: Remove partners for now

File: src/components/Navbar/mobile.js [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+    { name: "Careers", link: "/careers" }
+  ];
+  return (
+    <>
+      <AppBar position="static" className={classes.header}>
+        <Toolbar className={classes.toolBar}>
+          <IconButton
```
Comment (JoseChungFOS) [inline src/components/Navbar/mobile.js]: Remove this - it's covering the X button

File: src/components/Navbar/NavItemforMobile.js [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+import { Link } from "gatsby";
+const NavItemforMobile = ({ navItems }) => {
+  const classes = useStyles();
+
+  return (
+    <nav className={classes.navigation__nav}>
+      <List
+        component="nav"
+        aria-label="main mailbox folders"
+        className={classes.navigation__list}
```
Comment (JoseChungFOS) [inline src/components/Navbar/NavItemforMobile.js]: See note below on `navigation__list` CSS

File: src/components/Navbar/style.js [hunk 11]
```diff
@@ -0,0 +1,195 @@
...
+    maxHeight: "100vh",
+    position: "fixed",
+    top: "0",
+    left: "0",
+    zIndex: "1011",
+    opacity: "0",
+    width: "0",
+    transition: "all 0.8s"
+  },
+  navigation__list: {
```
Comment (JoseChungFOS) [inline src/components/Navbar/style.js]: As it is now, the list appears in the middle of the screen, with a large gap above it. I'm not quite understanding the CSS here, but if you remove all three of these it seems to fit perfectly
```
    top: "50%",
    left: "50%",
    transform: "translate(-50%, -50%)",
```

File: src/components/Navbar/opendItem.js [hunk 12]
```diff
@@ -0,0 +1,58 @@
+import React from "react";
```
Comment (JoseChungFOS) [inline src/components/Navbar/opendItem.js]: These work nicely. I think we might add a little transition to make it smoother in the future, but for now this is perfect

File: src/components/Navbar/HeaderNav.js (hunk 1 above)
Comment (fadeomar) [reply src/components/Navbar/HeaderNav.js]: Done

File: src/components/Navbar/HeaderNav.js (hunk 2 above)
Comment (fadeomar) [reply src/components/Navbar/HeaderNav.js]: Done

File: src/components/Navbar/HeaderNav.js (hunk 3 above)
Comment (fadeomar) [reply src/components/Navbar/HeaderNav.js]: Done

File: src/components/Navbar/HeaderNav.js (hunk 4 above)
Comment (fadeomar) [reply src/components/Navbar/HeaderNav.js]: Done

File: src/components/Navbar/HeaderNav.js (hunk 6 above)
Comment (fadeomar) [reply src/components/Navbar/HeaderNav.js]: good point thanks Jose

File: src/components/Navbar/HeaderNav.js (hunk 5 above)
Comment (fadeomar) [reply src/components/Navbar/HeaderNav.js]: Done

File: src/components/Navbar/index.js (hunk 7 above)
Comment (fadeomar) [reply src/components/Navbar/index.js]: Done

File: src/components/Navbar/mobile.js (hunk 8 above)
Comment (fadeomar) [reply src/components/Navbar/mobile.js]: Done

File: src/components/Navbar/mobile.js (hunk 9 above)
Comment (fadeomar) [reply src/components/Navbar/mobile.js]: Done

File: src/components/Navbar/NavItemforMobile.js (hunk 10 above)
Comment (fadeomar) [reply src/components/Navbar/NavItemforMobile.js]: Done :)

File: src/components/Navbar/opendItem.js (hunk 12 above)
Comment (fadeomar) [reply src/components/Navbar/opendItem.js]: alright Cool

File: src/components/Navbar/style.js (hunk 11 above)
Comment (fadeomar) [reply src/components/Navbar/style.js]: Done , thanks for that

File: src/components/Navbar/index.js [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+
+import AcountabilityLogo from "../../img/logo.svg";
+
+import HeaderNav from "./HeaderNav";
+import useStyles from "./style";
+
+export default function ButtonAppBar() {
+  const classes = useStyles();
+  const theme = useTheme()
+  return useMediaQuery("(min-width:960px)") ? (
```
Comment (JoseChungFOS) [inline src/components/Navbar/index.js]: On my screen, there's about 10px where the alignment just goes crazy - looks like 967px should be the breakpoint

```suggestion
  return useMediaQuery("(min-width:967px)") ? (
```

File: src/components/Navbar/mobile.js (hunk 9 above)
Comment (JoseChungFOS) [reply src/components/Navbar/mobile.js]: The logo is still appearing as a block above the X button on mobile

Comment (JoseChungFOS) [reply src/components/Navbar/mobile.js]: image

File: src/components/Navbar/style.js [hunk 14]
```diff
@@ -0,0 +1,200 @@
...
+  root: {
+    flexGrow: 1,
+    background: "#fff"
+  },
+  menuButton: {
+    marginRight: theme.spacing(2),
+    padding: 0
+  },
+  toolBar: {
+    paddong: 0
```
Comment (JoseChungFOS) [inline src/components/Navbar/style.js]:
```suggestion
    padding: 0
```

File: src/components/Navbar/mobile.js [hunk 15]
```diff
@@ -0,0 +1,48 @@
...
+    { name: "Advisory", link: "/advisory" },
+    { name: "Standards", link: "/standards" },
+    { name: "Insights", link: "/insights" },
+    { name: "Case Studies", link: "/case-studies" },
+    { name: "About", link: "/about" },
+    { name: "Careers", link: "/careers" }
+  ];
+  return (
+    <>
+      <AppBar position="static" className={classes.header}>
```
Comment (JoseChungFOS) [inline src/components/Navbar/mobile.js]: This header isn't supposed to appear in the expanded mobile view - it's supposed to look like this

image

But it looks like this

Selection_584

Can you set it to not display if the mobile nav is open / the checkbox is checked?

Personally I'd probably store the status in state (e.g. `const [expanded, setExpanded] = useState(false)`) but if you can do it with pure CSS then even better!

Comment (fadeomar) [reply src/components/Navbar/mobile.js]: It's working very well in my browser, can you tell me please how do you check this? what browser do you use or what type of mobile

File: src/components/Navbar/index.js (hunk 13 above)
Comment (fadeomar) [reply src/components/Navbar/index.js]: Done

File: src/components/Navbar/style.js (hunk 14 above)
Comment (fadeomar) [reply src/components/Navbar/style.js]: Done

**Pull request merged**